packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/TaskNavButtons.js

```diff
@@ -44,11 +46,21 @@ export default function TaskNavButtons (props) {
           areAnnotationsNotPersisted={props.areAnnotationsNotPersisted}
           onClick={props.goToPreviousStep}
         />}
+      <DoneAndTalkButton
+        completed={props.completed}
+        demoMode={props.demoMode}
+        flex='grow'
+        goldStandardMode={goldStandardMode}
+        onClick={props.onSubmit}
+        disabled={props.waitingForAnswer}
+        projectSlug={props.projectSlug}
+        subjectId={props.subjectId}
+      />
```

Comment:
These are a lot of props for a simple button.

Reply:
`projectSlug` and `subjectId` was cruft since I added a connected container component for the `DoneAndTalkButton` itself. This should be cleaned up now.